test/mongoShell.test.ts

```diff
@@ -51,6 +57,9 @@ suite("MongoShell", function (this: Mocha.Suite): void {
     } else if (!fse.existsSync(mongodPath)) {
         console.log(`Couldn't find mongo.exe at ${mongoPath} - skipping MongoShell tests`);
         testsSupported = false;
+    } else {
+        // Prevents code 100 error: https://aka.ms/AA8o5qm
+        await fse.ensureDir('D:\\data\\db\\');
```

Comment:
Weird to me that this is "D", but above the paths are "C"

Reply:
Mongo looked for that path on the D drive for whatever reason. Since the general idea from the stack overflow link was right I included it